app/scripts/modules/ecs/src/serverGroup/configure/wizard/CloneServerGroup.ecs.controller.js

```diff
@@ -184,6 +185,13 @@ module(ECS_SERVERGROUP_CONFIGURE_WIZARD_CLONESERVERGROUP_ECS_CONTROLLER, [
       processCommandUpdateResult($scope.command.regionChanged(serverGroupCommand));
     }
 
+    function changeCluster(){
```

Comment:
This controller doesn't actually load the UI. It just copies the properties of the previous server group to the new one. You shouldn't need to change any logic in here.